<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>619</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	/*background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 0px;
	margin: 5px;*/
	height: 40px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="headerLabel">
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #464e56
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Chatbox</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="sceneWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loadingPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="loadingPage_loading_label">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="loadingStatusLabel">
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="loadingErrorLabel">
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="welcomePage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>30</number>
          </property>
          <item>
           <layout class="QFormLayout" name="login_formLayout">
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QLabel" name="loginLabel">
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QFormLayout" name="formLayout_6">
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <property name="verticalSpacing">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>200</number>
              </property>
              <item row="1" column="1">
               <widget class="QLineEdit" name="login_UsernameEdit">
                <property name="maxLength">
                 <number>30</number>
                </property>
                <property name="placeholderText">
                 <string>Enter your username ...</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="login_PasswordEdit">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Enter your password ...</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="login_Button">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="login_statusLabel">
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>13</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_8">
            <property name="bottomMargin">
             <number>100</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="welcomePage_loading_label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="or_label">
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="text">
               <string>OR</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="register_formLayout">
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QLabel" name="registrationLabel">
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>30</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Register</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QFormLayout" name="registrationLayout">
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <property name="verticalSpacing">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>200</number>
              </property>
              <item row="1" column="1">
               <widget class="QLineEdit" name="registration_UsernameEdit">
                <property name="maxLength">
                 <number>30</number>
                </property>
                <property name="placeholderText">
                 <string>Enter your username ...</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="registration_PasswordEdit">
                <property name="maxLength">
                 <number>64</number>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Enter your password ...</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="registration_Button">
                <property name="text">
                 <string>Register</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="registration_statusLabel">
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="leftLayout" native="true">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QGroupBox" name="headerBox">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>150</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Status</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Online/Offline/DND</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="chat_button">
                    <property name="text">
                     <string>Chats</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="contacts_button">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Contacts</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <layout class="QGridLayout" name="left_bottom_layout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QStackedWidget" name="chat_contacts_stackedWidget">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="chats_page">
                  <layout class="QGridLayout" name="gridLayout_10">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QScrollArea" name="scrollArea">
                     <property name="widgetResizable">
                      <bool>true</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                     <widget class="QWidget" name="chats_scroll_area">
                      <property name="geometry">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>98</width>
                        <height>28</height>
                       </rect>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_12">
                       <property name="sizeConstraint">
                        <enum>QLayout::SetNoConstraint</enum>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <item row="0" column="0">
                        <layout class="QGridLayout" name="chats_grid_layout">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                        </layout>
                       </item>
                      </layout>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="contacts_page">
                  <layout class="QGridLayout" name="gridLayout_11">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QScrollArea" name="scrollArea_2">
                     <property name="widgetResizable">
                      <bool>true</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignHCenter|Qt::AlignTop</set>
                     </property>
                     <widget class="QWidget" name="contacts_scroll_area">
                      <property name="geometry">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>98</width>
                        <height>28</height>
                       </rect>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_14">
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <item row="0" column="0">
                        <layout class="QGridLayout" name="contacts_grid_layout">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                        </layout>
                       </item>
                      </layout>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget_2">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
